tensorflow/lite/micro/tools/make/ext_libs/ethosu.inc

```diff
@@ -1,19 +1,31 @@
 ifneq ($(filter ethos-u,$(ALL_TAGS)),)
-    # Don't want -lm flag
-    MICROLITE_LIBS :=
+    MICROLITE_LIBS := $(filter-out -lm,$(MICROLITE_LIBS))
```

Comment:
Add short comment to why we filter out the -lm flag

Reply:
Seems like it is not longer needed, will remove it.

Reply:
@mansnils Any update on this PR? Please. Thanks!

Reply:
It might still be needed. So I am adding a comment. I think it more belong in a tools/make/target/makefile but there is none for this yet so let's keep it here for now.

Reply:
Done

Reply:
Will https://github.com/tensorflow/tensorflow/pull/43726 make the filtering out here not be needed anymore? Let's get this PR merged since its been sitting for a long time but then circle back and have the exclusion in a single place.

Reply:
As long as the target in https://github.com/tensorflow/tensorflow/pull/43726 is used I guess the filtering out here is not needed anymore. As said let's revisit this later.